support/sl-post-hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Please be sure to add this to your documentation and/or what's this for exactly? This seems out of place.